mypy/semanal.py

```diff
@@ -2366,11 +2374,19 @@ def fail_typeddict_arg(self, message: str,
 
     def build_typeddict_typeinfo(self, name: str, items: List[str],
                                  types: List[Type]) -> TypeInfo:
-        mapping_value_type = join.join_type_list(types)
         fallback = (self.named_type_or_none('typing.Mapping',
```

Comment:
My previous comment was just my interpretation of the status quo. As far as I am concerned, I would prefer fallback to be just ``Dict`` (this would be most precise and most similar to how tuples and named tuples work).